modules/tiles/src/tileset/tile-3d.ts

```diff
@@ -303,6 +309,17 @@ export class Tile3D {
     return this._screenSpaceError;
   }
 
+  /**
+   * Get bounding box in cartographic coordinates
+   * @returns [min, max] each in [longitude, latitude, altitude]
+   */
+  get boundingBox(): CartographicBounds {
```

Comment:
The naming is mixed up with "oriented bounding box". I would prefer `extent` here . I3S before 1.7 version had extent2d https://github.com/Esri/i3s-spec/blob/master/docs/1.7/3DSceneLayer.cmn.md . From 1.8 there is 'fullExtent' with z coordinate -  https://github.com/Esri/i3s-spec/blob/master/docs/1.8/3DSceneLayer.cmn.md